packages/ui/src/Slider/index.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { Slider } from '.';
+import { PenumbraUIProvider } from '../PenumbraUIProvider';
+
+describe('<Slider />', () => {
+  it('renders correctly', () => {
```

Comment:
thought: that's not a meaningful test. UI interactions could be properly tested.

suggestion: for this test, you can render the component, simulate a click to some point of the slider, mock the `onChange` function and expect the correct value to be passed to it.